### PR TITLE
Apache WildFly Receiver Metrics

### PR DESCRIPTION
**Pull request by StefanKurek**
Adds ability to collect Apache WildFly metrics to ops-agent.

[Scope Doc](https://docs.google.com/document/d/1zK4x0egXpmSCXL-nDed0e323LrvHbz1qxp1ap-3Dlaw/edit)

### CODE REVIEW

File: confgenerator/testdata/invalid/linux/metrics-receiver_wildfly_additional_jar_does_not_exist/input.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+metrics:
+  receivers:
+    wildfly:
+      type: wildfly
+      username: usr
+      password: pas
+      collection_interval: 30s
+      additional_jars:
+      - /does_not_exit.jar
```
Comment (jefferbrecht) [inline confgenerator/testdata/invalid/linux/metrics-receiver_wildfly_additional_jar_does_not_exist/input.yaml:23]:
```suggestion
      - /does_not_exist.jar
```

**Review by jefferbrecht — COMMENTED** [review]
LGTM, just some minor comments + we'll wait for the integration tests too.

File: docs/wildfly.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
+# `wildfly` Metrics Receiver
+
+The `wildfly` metrics receiver can fetch stats from a WildFly server's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html).
+
+## Prerequisites
+To expose JMX endpoint remotely, you should also set the `jboss.bind.address.management` system property. By default, this property is set in the WildFly's config xml file. The default WildFly installation requires no JMX authentication with JMX exposed locally on 127.0.0.1:9990.
```
[the remark this answers is not in the document]
Comment (StefanKurek) [reply docs/wildfly.md]: @jefferbrecht fixed

**Comment by StefanKurek** [comment]
@jefferbrecht fixed changes, kokoro tests previously passed. so all should be good now

**Review by jefferbrecht — APPROVED** [review]

**Pull request merged**